user-service/src/main/java/rs/edu/raf/banka1/model/CurrentAccount.java

```diff
@@ -0,0 +1,25 @@
+package rs.edu.raf.banka1.model;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+public class CurrentAccount {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String accountNumber;
+    private Long ownerId;
+    private Double balance;
+    private Double availableBalance;
+    private Long createdByAgentId;
+    private Integer creationDate;
+    private Integer expirationDate;
+    private String currency;
+    private String accountStatus;
+    private String subtypeOfAccount;
+    private String accountMaintenance;
```

Comment:
Specification example was 255.00,00 RSD so I suppose that's the reason this attribute is set to String. Monthly fee will be withdrawn from the user's account every month so I think it's better to change attribute type to Integer for example, because, if saved as String, monthly fee will need to be parsed from the string which I don't think is necessary.

Reply:
Maybe it's better to set to Long just in case?

Reply:
@savatarko noone is that rich!

Reply:
fixed!